int32     id
Point2d   center
Point2d   direction
Point2d[] corners
Point2d[] path
Point2d   actual_point
Point2d   next_point
int32     angle_to_actual_point
int32     actual_angle
int32[]   sector

bool      path_created
bool      on_finish
bool      move
bool      rotation
bool      stop


